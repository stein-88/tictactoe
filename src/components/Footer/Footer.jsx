import React from 'react'
import schema from './data'
import s from './Footer.scss'

const Footer = () => {
	const { text } = schema
	return (
		<footer>
			<section id="footer" className={`py-5 ${s.bgblack}`}>
				<p className="py-5 text-center text-white">{text}</p>
			</section>
		</footer>
	)
}

export default Footer
