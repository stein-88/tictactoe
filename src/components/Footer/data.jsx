const schema = {
  text: 'Â© 2022 Wanderson Santos. All Rights Reserved.',
};

export default schema;
