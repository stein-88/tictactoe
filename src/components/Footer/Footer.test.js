import React from 'react'
import renderer from 'react-test-renderer'
import Footer from './Footer'

describe('Test on Footer Component', () => {
    const compFooter = renderer.create(<Footer />)
    test('Sould render with no props', () => {
        const jsonFooter = compFooter.toJSON()
        expect(jsonFooter).toMatchSnapshot()
    })
    test('Sould have a text', () => {
        const Result = compFooter.root.findAllByProps({ className: "py-5 text-center text-white" })
        expect(Result).toHaveLength(1)
    })
})