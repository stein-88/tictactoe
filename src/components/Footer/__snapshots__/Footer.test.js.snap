// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test on Footer Component Sould render with no props 1`] = `
<footer>
  <section
    className="py-5 bgblack"
    id="footer"
  >
    <p
      className="py-5 text-center text-white"
    >
      Â© 2022 Wanderson Santos. All Rights Reserved.
    </p>
  </section>
</footer>
`;
